code/game/objects/items/crayons.dm

```diff
@@ -492,6 +492,11 @@
 	reagent_contents = list(/datum/reagent/consumable/nutriment = 0.5,  /datum/reagent/colorful_reagent/powder/white/crayon = 1.5)
 	dye_color = DYE_WHITE
 
+/obj/item/toy/crayon/white/chalk
+	name = "detective's chalk"
```

Comment:
Currently, when this is examined, it will say 'That's a detective's chalk.' What are your thoughts on just 'That's detective's chalk'?
```suggestion
	name = "\proper detective's chalk"
```